frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -177,7 +177,8 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
                     !_.isEmpty(errors) ||
                     !_.isEmpty(status?.tasks) ||
                     !_.isEmpty(status?.[STATUS_KEY_NAME_ERROR]) ||
-                    formData.tasks.length === 0
+                    formData.tasks.length === 0 ||
+                    formData.tasks.filter((task) => task?.metadata?.installing).length > 0
```

Comment:
nit:
```suggestion
                    formData.tasks.filter((task) => task.metadata?.installing).length > 0
```

Reply:
Here the metadata is optional field, that is managed by us to show the installing decorator.